debian/control

```diff
@@ -1,9 +1,9 @@
 Package: thelounge
-Version: 3.0.0-pre.6-1
+Version: 3.0.0-pre.6-2
 Section: net
 Priority: optional
-Architecture: all
-Depends: nodejs (>= 6), bash, sqlite3
+Architecture: amd64
```

Comment:
Will that mean RPi < 3 will not be able to run this?
I know it doesn't mean non-amd64 were running fine before, but I'm just making sure I get what this means (I never paid too much attention to this, but I run a RPi 2 that I would love to use this .deb with)

Reply:
Yeah right now we can't provide automatic ARM binaries because Travis doesn't have infrastructure to run there (probably can via some manual virtualization..).

Reply:
Also, Node 10 (nor nodesource) no longer provide i386 builds.

Reply:
Does this mean after this PR the deb is not going to work for _all_ Raspberry Pis running Raspbian, or just RPis 2 and earlier?

Reply:
Did it work before on Raspbian? I've only seen people installing Lounge via npm there.

Reply:
Almost certain the deb works with v2.7.1 yeah.